diff -uNr android-mtk-3.18/tools/perf/Makefile android-mtk-3.18.mod/tools/perf/Makefile
--- android-mtk-3.18/tools/perf/Makefile	2024-04-14 21:55:48.795992144 +0800
+++ android-mtk-3.18.mod/tools/perf/Makefile	2024-04-14 21:57:46.295992060 +0800
@@ -54,8 +54,10 @@
   @printf '  BUILD:   Doing '\''make \033[33m-j'$(JOBS)'\033[m'\'' parallel build\n'
 endef
 
+export CC
+
 define make
-  @$(MAKE) -f Makefile.perf --no-print-directory -j$(JOBS) O=$(FULL_O) $(SET_DEBUG) $@
+  @$(MAKE) -f Makefile.perf CC=$(CC) --no-print-directory -j$(JOBS) O=$(FULL_O) $(SET_DEBUG) $@
 endef
 
 #
