diff -uNr android-mtk-3.18/tools/lib/api/fs/fs.c android-mtk-3.18.mod/tools/lib/api/fs/fs.c
--- android-mtk-3.18/tools/lib/api/fs/fs.c	2024-04-14 15:00:48.929000000 +0800
+++ android-mtk-3.18.mod/tools/lib/api/fs/fs.c	2024-04-14 22:59:56.989999545 +0800
@@ -5,6 +5,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 #include <sys/vfs.h>
 
@@ -75,9 +76,20 @@
 
 	if (statfs(fs, &st_fs) < 0)
 		return -ENOENT;
-	else if (st_fs.f_type != magic)
+
+#if defined(__arm__)
+#define MAGIC_TYPE(a)  ((uint32_t)(a & 0xffffffff))
+#elif defined(__aarch64__)
+#define MAGIC_TYPE(a)  ((uint64_t)(a & 0xffffffffffffffff))
+#else
+#define MAGIC_TYPE(a)  (a)
+#endif
+
+	else if (st_fs.f_type != (MAGIC_TYPE(magic)))
 		return -ENOENT;
 
+#undef MAGIC_TYPE
+
 	return 0;
 }
 
